packages/perf/android/src/main/java/io/invertase/firebase/perf/ScreenTrace.java

```diff
@@ -0,0 +1,227 @@
+package io.invertase.firebase.perf;
+
+/**
+ * Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
where did this code come from? You must sign the CLA here which asserts that it is your code, but the copyright is Google :thinking: ?

Reply:
99.99% of this code is Google's which they offered to the community. I modified it slightly for our use cases here.

Reply:
I am happy to completely refactor this code if that's necessary for licensing. It looks like we're making plenty of modifications to the file anyways.

Reply:
I think the Apache license is fine, I was just curious - this won't block